src/chrome/content/rules/E-Hentai.org.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+	Cloudflare error:
+		- g
+
+	Timeout:
+		- monitor
+		- shingata
+		- test
+
+	Mismatched:
+		- wiki
+-->
+<ruleset name="E-Hentai.org">
+	<target host="e-hentai.org" />
+	<target host="www.e-hentai.org" />
+	<target host="api.e-hentai.org" />
+	<target host="forum.e-hentai.org" />
+	<target host="forums.e-hentai.org" />
+	<target host="forumtest.e-hentai.org" />
+	<target host="g.e-hentai.org" />
+	<target host="hv.e-hentai.org" />
+	<target host="lofi.e-hentai.org" />
+	<target host="r.e-hentai.org" />
+	<target host="repo.e-hentai.org" />
+	<target host="ul.e-hentai.org" />
+	<target host="ulv.e-hentai.org" />
+	<target host="upload.e-hentai.org" />
+	<target host="wiki.e-hentai.org" />
+	<target host="xml.e-hentai.org" />
+
+	<rule from="^http://g\.e-hentai\.org/" to="https://e-hentai.org/" />
+	<rule from="^http://wiki\.e-hentai\.org/" to="https://ehwiki.org/" />
```

Comment:
These two rules may be removed, but both of them mirror website's own 301 redirects.